pkgs/development/tools/gojsontoyaml/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "gojsontoyaml";
+  version = "3697ded27e8cfea8e547eb082ebfbde36f1b5ee6";
```

Comment:
Please use a semver version or the date of the commit `unstable-XXXX-XX-XX`.

Reply:
Thanks for the suggestion, I was really unsure about that point.

Reply:
Done, using unstable-YYYY-MM-DD as version with latest commit as ref.